crates/command_palette/src/command_palette.rs

```diff
@@ -37,6 +39,10 @@ pub struct CommandPalette {
     picker: View<Picker<CommandPaletteDelegate>>,
 }
 
+lazy_static! {
+    static ref STRIP_UNNECESSARY_WHITESPACES: Regex = Regex::new(r"\s+").unwrap();
```

Comment:
I'd prefer a name that describes what the regex matches, not what's being done when it's used:

```suggestion
    static ref CONSECUTIVE_WHITESPACES: Regex = Regex::new(r"\s+").unwrap();
```

Reply:
Oh it's fine @mrnugget. I'm doing it now. Let's close this one out properly. 😄